server/eb/check_config.py

```diff
@@ -19,8 +19,8 @@ def main():
     args = parser.parse_args()
 
     app_config = AppConfig()
-    app_config.update_from_config_file(args.config_file)
     try:
+        app_config.update_from_config_file(args.config_file)
```

Comment:
Did you intend to have code changes in a readme update pr?

Reply:
I did it intentionally for convenience.    Although certainly it could have been two PRs.   The code changes were to clean up the old way of handling secrets, and document was updated to reflect those changes, so it was related.